content/en/docs/reference/command-line-tools-reference/feature-gates/default-host-network-ports-in-pod-templates.md

```diff
@@ -10,9 +10,12 @@ stages:
     defaultValue: false
     fromVersion: "1.28"  
 ---
-Changes when the default value of
-`PodSpec.containers[*].ports[*].hostPort`
-is assigned. The default is to only set a default value in Pods.
+This feature gate controls the point at which a default value for
+`.spec.containers[*].ports[*].hostPort`
+is assigned, for Pods. The default since Kubernetes v1.28 is to only set a default
```

Comment:
```suggestion
is assigned, for Pods using `hostNetwork: true`. The default since Kubernetes v1.28 is to only set a default
```

Reply:
it seems only appplies to hostNetwork https://github.com/kubernetes/kubernetes/pull/117696/